IFC4x3/Properties/w/WorkingState_2qD57tYLb86QIkv8vmjv3f/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="WorkingState_2qD57tYLb86QIkv8vmjv3f" Name="WorkingState" UniqueId="b43451f7-8959-4819-a4ae-e48e70b790e9" PrimaryDataType="IfcLabel" />
```

Comment:
Not sure about this property.  If adopted, documentation should be made more generic so it can be reused.  Should it be an enumeration?